src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeed.cs

```diff
@@ -204,7 +207,7 @@ private string GenerateRootServiceIndex(string redirectUrl)
                 writer.WriteStartArray();
                 writer.WriteStartObject();
                 writer.WritePropertyName("@id");
-                writer.WriteRawValue($"\"{FeedContainerUrl}/{redirectUrl}/\"");
+                writer.WriteRawValue($"\"{FeedUrl}\"");
```

Comment:
Why wouldn't FeedUrl completely replace FeedContainerUrl? 

This looks like it is writing to the root index.json file but what about the individual package index.json files?

Reply:
We still need the FeedContainerUrl to make calls to the storage account(createcontainer, checkifblobexists, etc.) I've left it in there for now.

the individual package index.json don't have urls in them, just the versions available for that package.